requirements-python3_6/dev-requirements.txt

```diff
@@ -119,7 +116,7 @@ mako==1.0.7
 markdown==2.2.1
 markupsafe==1.1.1
 mock==2.0.0
-modernize==0.6
+modernize==0.7
```

Comment:
@nickpell fyi that modernize upgraded. they don't have an updated changelog though

Reply:
good to know, thanks